js/google_ima.js

```diff
@@ -1245,6 +1240,11 @@ require("../html5-common/js/utils/utils.js");
             if(ad.isLinear())
             {
               _linearAdIsPlaying = true;
+              if (this.savedVolume >= 0)
```

Comment:
Yep if the ad is started, _linearAdIsPlaying is true and the setVolume will call the IMA API rather than saving it.